src/main/java/core/basesyntax/service/db/StorageServiceImpl.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.service.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class StorageServiceImpl implements StorageService {
+    private final Map<String, Integer> storage = new HashMap<>();
```

Comment:
You should follow the encapsulation principle and make your storage private. It's a bad practice to make your class exposed.